service/src/resources/hive-webapps/hiveserver2/loginForm.jsp

```diff
@@ -0,0 +1,34 @@
+<%@ page contentType="text/html;charset=UTF-8" %>
```

Comment:
this file also needs an apache header I supposed, similarly to 
https://github.com/apache/hive/blob/master/service/src/resources/hive-webapps/hiveserver2/hiveserver2.jsp

Reply:
Done

Reply:
ack